.github/workflows/main.yml

```
@@ -72,31 +78,20 @@ jobs:
-      - name: Build carbon-apimgt without tests, skip AspectJ, skip portal builds
-        run: mvn clean install --file pom.xml -Dmaven.test.skip=true -Dskip.aspectj=true -Dnpm.build.skip=true
-      - name: Get carbon.apimgt.version
-        run: mvn help:evaluate -Dexpression=project.version -q -DforceStdout > ../CARBON_APIMGT_VERSION_FILE
-      - name: Print carbon.apimgt.version
-        run: |
-          echo $(cat ../CARBON_APIMGT_VERSION_FILE)
+          key: ${{ runner.os }}-m2-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-m2-
+      - name: Print carbon.apimgt.version from build-carbon
+        run: echo "Carbon version: ${{ needs.build-carbon.outputs.carbon_version }}"
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 94: mapping values are not allowed in this context

(syntax-check)